tests/integration.rs

```diff
@@ -51,8 +51,9 @@ fn test_keys_pos() -> Result<()> {
     Ok(())
 }
 
-#[test]
-fn test_test_helper_version() -> Result<()> {
+#[cfg(feature = "experimental-api")]
+#[cfg_attr(feature = "experimental-api", test)]
+fn test_helper_version() -> Result<()> {
```

Comment:
Why #[test] is not needed?

Reply:
Oh you put it on the `cfg_attr` :+1: , So we we also need `#[cfg(feature = "experimental-api")]`?